src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensParticle.java

```diff
@@ -75,22 +82,81 @@ public void run() {
 		enabled = true;
 	}
 
+
 	@Override
 	public void run() {
-		for (NPC npc : npcs) {
-			if (npc == null) {
-				continue;
-			}
-			Entity e = npc.getEntity();
-			if (e == null) {
-				continue;
-			}
-			Location loc = e.getLocation().clone();
-			loc.setPitch(-90);
-			Compatibility.getEffectManager().start(name, section, loc);
-		}
+	    // Bukkit.getLogger().info("Particle: RUN:89: Online Players: " + Bukkit.getOnlinePlayers().size() + "NPC:" + npcs.size());
+        for (NPC npc : npcs) {
+            if (npc == null) {
+                continue;
+            }
+            Entity e = npc.getEntity();
+            if (e == null) {
+                continue;
+            }
+            // Bukkit.getLogger().info("NPC Name: " + npc.getName());
+            Location loc = e.getLocation().clone();
+            loc.setPitch(-90);
+            // Compatibility.getEffectManager().start(name, section, loc);
+        }
+
+
+        Config.getPackages().forEach((n, c) -> activateEffects(n, c));
 	}
 
+    void activateEffects(String name, ConfigPackage cfgPack){
```

Comment:
This would be better off being private, rather than limited to the whole package.